packages/components/pie-checkbox/test/component/pie-checkbox.spec.ts

```diff
@@ -310,6 +316,75 @@ test.describe('PieCheckbox - Component tests', () => {
                 expect(isValid).toBe(true);
             });
         });
+
+        test.describe('assistiveText', () => {
+            test('should not render the assistive text component if the prop is not provided', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieCheckbox, {});
+
+                // Act
+                const assistiveText = page.locator(assistiveTextSelector);
+
+                // Assert
+                expect(assistiveText).not.toBeVisible();
+            });
+
+            test('should apply the "default" variant attribute if no status is provided', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieCheckbox, {
+                    props: {
+                        assistiveText: 'Assistive text',
+                    } as PieCheckbox,
+                });
+
+                // Act
+                const assistiveText = page.locator(assistiveTextSelector);
+
+                // Assert
+                expect(assistiveText).toBeVisible();
+                expect(await assistiveText.getAttribute('variant')).toBe('default');
+                expect(assistiveText).toHaveText('Assistive text');
+            });
+
+            test.describe('Assistive text: Status', () => {
+                statusTypes.forEach((status) => {
+                    test(`should render the assistive text component with the ${status} variant`, async ({ mount, page }) => {
+                        // Arrange
+                        await mount(PieCheckbox, {
+                            props: {
+                                assistiveText: 'Assistive text',
+                                status,
+                            } as PieCheckbox,
+                        });
+
+                        // Act
+                        const assistiveText = page.locator(assistiveTextSelector);
+
+                        // Assert
+                        expect(assistiveText).toBeVisible();
+                        expect(assistiveText).toHaveAttribute('variant', status);
+                        expect(assistiveText).toHaveText('Assistive text');
+                    });
+                });
+            });
+
+            test.describe('Assistive test ID attribute', () => {
+                test('should contain an ID associated the checkbox element for a11y', async ({ mount, page }) => {
+                    // Arrange
+                    await mount(PieCheckbox, {
+                        props: {
+                            assistiveText: 'Assistive text',
+                        } as PieCheckbox,
+                    });
+
+                    // Act
+                    const assistiveText = page.locator(assistiveTextSelector);
+
+                    // Assert
+                    await expect(assistiveText).toHaveAttribute('id', 'assistive-text');
```

Comment:
Do we also want to check that the checkbox has `aria-describedby` with the same value?

Reply:
I updated this test case and added 2 more tests for `assistive text` and `aria-describedby`: 

1. when `assistiveText` is not defined -> should not render the attribute
2. when `assistiveText` is defined -> should render the attribute correctly with the correct value